conda/pytorch-nightly/bld.bat

```diff
@@ -65,6 +65,20 @@ if "%CUDA_VERSION%" == "80" (
     set "CUDAHOSTCXX=%VS140COMNTOOLS%\..\..\VC\bin\amd64\cl.exe"
 )
 
+:: randomtemp is used to resolve the intermittent build error related to CUDA.
+:: code: https://github.com/peterjc123/randomtemp
+:: issue: https://github.com/pytorch/pytorch/issues/25393
+::
+:: Previously, CMake uses CUDA_NVCC_EXECUTABLE for finding nvcc and then
+:: the calls are redirected to sccache. sccache looks for the actual nvcc
+:: in PATH, and then pass the arguments to it.
+:: Currently, randomtemp is placed before sccache (%TMP_DIR_WIN%\bin\nvcc)
+:: so we are actually pretending sccache instead of nvcc itself.
+curl -kL https://github.com/peterjc123/randomtemp/releases/download/v0.3/randomtemp.exe --output %SRC_DIR%\tmp_bin\randomtemp.exe
+set RANDOMTEMP_EXECUTABLE=%SRC_DIR%\tmp_bin\nvcc.exe
+set CUDA_NVCC_EXECUTABLE=%SRC_DIR%\tmp_bin\randomtemp.exe
```

Comment:
Shouldn't this be other way around?

Reply:
Could you please give an example?

Reply:
Currently, we can only workground this issue by retrying it several times.

Reply:
Why `RANDOMTEMP_EXECUTABLE` is points to `nvcc` and `CUDA_NVCC_EXECUTABLE` poitns to `randomtemp.exe`?

Reply:
Because `randomtemp.exe` acts like a shim exectuable, it just passes the argument to the actual program, which is specified by the variable `RANDOMTEMP_EXECUTABLE`.

Reply:
Yea I agree with @malfet this does seem a bit confusing when you look at it.

Maybe it'd be better to add a comment here explaining why they point where they point to.

Reply:
I think I understand how it works, but naming of variables is a bit confusing, but indeed it matches one on master